.github/workflows/crowdin.yml

```diff
@@ -13,31 +13,55 @@ concurrency:
 
 jobs:
   synchronize-with-crowdin:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    container: swift:5.10.1
 
-    steps:      
+    steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Crowdin action
         uses: crowdin/github-action@v2
         with:
           # For more info: https://github.com/crowdin/github-action/blob/master/action.yml
-          project_id: ${{ secrets.CROWDIN_PROJECT_ID }}          
+          project_id: ${{ secrets.CROWDIN_PROJECT_ID }}
           token: ${{ secrets.CROWDIN_API_TOKEN }}
           config: crowdin.yml
 
           upload_sources: true
           download_translations: true
           upload_translations: false
-
-          create_pull_request: true
+          create_pull_request: false
+          push_translations: true
           localization_branch_name: chore/update-localization
           commit_message: "chore: Update localization"
-          pull_request_title: "chore: Update localization - no ticket"
-          pull_request_body: "This PR pulls in the latest localization translations from Crowdin."
           github_user_name: "zenkins"
           github_user_email: "iosx@wire.com"
-          pull_request_team_reviewers: "wireapp/ios"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Trim unused localizations from string catalogs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          find . -name "*.xcstrings" | tee string-catalogs.txt
+          xargs swift run --package-path ./scripts TrimStringCatalogs < string-catalogs.txt
+          xargs git add < string-catalogs.txt
```

Comment:
why do we save this file?

Reply:
Just to not run the find command twice.
It will be discarded then.